### ISSUE TITLE
Firefox 

### ISSUE THREAD

**Issue by ghost** [mrieck/productivityowl#3]
Please add this as a Firefox add-on.
From my quick testing, it seems to be working just fine.

However, there is this warning.
<img width="519" alt="image" src="https://user-images.githubusercontent.com/103877079/166932569-f1295acf-addd-4e6a-a932-6e05f7161ee8.png">
Thank you.

**Comment by Hostuu** [comment]
Yes, this would be awesome on gecko browsers! How did you install it?
